### PR TITLE
[prancible] upgrade the version of ansible

### LINKED ISSUE #4567: Maintenance playbook should run apt clean
We've had a couple of machines run out of space because of too many old apt package files.

PR #4562 has the correct fix, but the task currently fails. The `clean` option was added in Ansible Core 2.13, and we are still running and older version of Ansible with Ansible Core 2.12. So this issue is related to #3782.

### PR DESCRIPTION
**Pull request by kayiwa**
this updates our version of ansible to 9.1.0 

closes #4567

### CODE REVIEW

**Review by VickieKarasic — APPROVED** [review]
This looks good to me - thanks @kayiwa

**Review by acozine — CHANGES_REQUESTED** [review]
I'm eager to get Ansible upgraded, but I would like to see at least some Molecule tests run and succeed before merging this. It looks like our CI suite just skipped everything - they all seem to say "successful" but in under 60 seconds each, which doesn't sound like any tests actually ran. I'm going to review this as "Request changes" but the PR itself looks fine - the review is only intended to hold up merging until we can at least run the tests locally.

File: roles/tomcat9/defaults/main.yml
Comment (acozine) [inline roles/tomcat9/defaults/main.yml:7]: These vars are all named `tomcat8_` something - should we update them to be `tomcat9_`? I'm not clear if we're changing the existing role to install tomcat 9 instead of tomcat 8, or if we need to maintain both for a while, or what. But I find this confusing now, and I bet Future Me will find it even more confusing in future.

File: roles/tomcat9/handlers/main.yml
Comment (acozine) [inline roles/tomcat9/handlers/main.yml:2]: Here too, and in the service name below . . .

File: roles/tomcat9/meta/main.yml
Comment (acozine) [inline roles/tomcat9/meta/main.yml:3]: And here.

File: roles/tomcat9/meta/main.yml
Comment (acozine) [inline roles/tomcat9/meta/main.yml:5]: And here.

File: roles/tomcat9/molecule/default/converge.yml
Comment (acozine) [inline roles/tomcat9/molecule/default/converge.yml:15]: This one would be particularly challenging if we continue to have a role called `tomcat8`, since then the tomcat9 tests would actually be running the tomcat8 tasks . . .

File: roles/tomcat9/vars/main.yml
Comment (acozine) [inline roles/tomcat9/vars/main.yml:2]: 9

**Review by acozine — COMMENTED** [review]
This PR is doing a lot of great stuff! The Tomcat changes are a little confusing, and there's still a CI failure, but we're getting there.

File: roles/tomcat9/defaults/main.yml
Comment (kayiwa) [reply roles/tomcat9/defaults/main.yml:7]: we are replacing tomcat8. I was uncomfortable removing it because of the geoserver but will remove it with the next commit.

**Review by acozine — APPROVED** [review]
CI success, huzzah!

**Pull request merged**
